starter-code/js/index.js

```diff
@@ -31,12 +33,22 @@ function createItemNode(dataType, itemData){
 }
 
 function createNewItemRow(itemName, itemUnitPrice){
```

Comment:
you'll be creating as many elements as there are inside the new row (li/div/span...)